jar-connector/src/test/java/com/sixsq/slipstream/persistence/VmTest.java

```diff
@@ -36,8 +36,10 @@
 
```

Comment:
The tests have moved from the `java-persistence` module to the `java-connector` module. This means that `Vm.java` and `VmTest.java` are in completely different modules, which isn't a good practice.  I suspect this has been done to avoid a circular dependency between these modules.  However, rather than splitting tests would it be a better idea to move the Collector (which depends on persistence and the connectors) to a "neutral" location?